README.md

```diff
@@ -57,14 +57,19 @@ point.
     docker run --rm -it -v myvolume:/workdir crops/poky --workdir=/workdir
     ```
 
-  Let's discuss the options:
-  * **_--workdir=/workdir_**: This causes the container to start in the directory
+  Available options:
+  * **_--help_**: show help message and exit.
+  * **_--workdir=WORKDIR_**: This causes the container to start in the directory
     specified. This can be any directory in the container. The container will also use the uid and gid
     of the workdir as the uid and gid of the user in the container.
+  * **_--id=ID_**: uid and gid to use for the user inside the container. It should be in the form uid:gid.
 
-  This should put you at a prompt similar to:
+  The previous example should put you at a prompt similar to:
   ```
   pokyuser@3bbac563cacd:/workdir$
   ```
   At this point you should be able to follow the same instructions as described
-  in https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html
+  in https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html#building-your-image to build an image.
+
+  Note that the container should not be used to clone Poky or obtain any other meta-data, which must be done
+  _outside_ the container using common tools like Git, as stated in https://docs.yoctoproject.org/brief-yoctoprojectqs/index.html#use-git-to-clone-poky
```

Comment:
This is not true. The instructions above tell you to create your workspace and pass that in as a volume. If you pass in a directory that does not exist, docker will "conveniently" create it inside the container as root.

For instance, this series of commands works just fine (on Linux) and poky is cloned with my user permissions:

```
ttorling@chiron:/srv/build/ttorling$ mkdir crops-workspace
ttorling@chiron:/srv/build/ttorling$ docker run --rm -it -v /srv/build/ttorling/crops-workspace:/workdir crops/poky --workdir=/workdir
pokyuser@14be76de76a9:/workdir$ git clone https://git.yoctoproject.org/poky.git
Cloning into 'poky'...
remote: Enumerating objects: 646774, done.
remote: Counting objects: 100% (1691/1691), done.
remote: Compressing objects: 100% (425/425), done.
remote: Total 646774 (delta 1364), reused 1419 (delta 1260), pack-reused 645083
Receiving objects: 100% (646774/646774), 204.82 MiB | 3.45 MiB/s, done.
Resolving deltas: 100% (471021/471021), done.
pokyuser@14be76de76a9:/workdir$ exit
exit
ttorling@chiron:/srv/build/ttorling$ ls -la crops-workspace/
total 12
drwxrwxr-x  3 ttorling ttorling 4096 May 24 09:20 .
drwxr-xr-x 73 ttorling ttorling 4096 May 24 09:18 ..
drwxr-xr-x 12 ttorling ttorling 4096 May 24 09:21 poky
```

There are also instructions for [Windows](https://github.com/crops/docker-win-mac-docs/wiki/Windows-Instructions-%28Docker-Toolbox%29#create-the-samba-container) and [Mac OS](https://github.com/crops/docker-win-mac-docs/wiki/Mac-Instructions#create-the-volume) to use the samba container, because those native file systems are not compatible with Linux.

Reply:
On further review of the [original bug](https://bugzilla.yoctoproject.org/show_bug.cgi?id=11385), the issue being discussed there is the lack of the `repo` tool in the image. This has already been discussed and rejected https://github.com/crops/yocto-dockerfiles/pull/42

The solution is to build a derivative container with the tools you need for your specific use case.